### DISCUSSION TITLE
Move Kamal-related configuration to ./kamal/*

### DISCUSSION THREAD

**Discussion by igor-alexandrov** [basecamp/kamal#618] [Ideas]
Here (https://twitter.com/motdotla/status/1738589409119961343) @motdotla suggested that Kamal's configuration is currently too Rails-centric. I agree that keeping the deploy configuration in `./config/` looks correct only in Rails. Yes, Kamal was originally introduced as a Rails tool, but now it has a wider audience. What do you think about using `./kamal/*` as a default configuration folder for Kamal?

**Comment by dhh** [comment]
Don't think that adds any additional value even if you're not using Rails. That's still one more directory.
